src/components/v5/common/ActionSidebar/partials/Motions/steps/OutcomeStep/hooks.ts

```diff
@@ -0,0 +1,66 @@
+import { useMemo } from 'react';
+import { Id } from '@colony/colony-js';
+import { supportOption, opposeOption } from '../../consts';
+import { MotionVote } from '~utils/colonyMotions';
+import { ColonyMotion } from '~types';
+import { COLONY_TOTAL_BALANCE_DOMAIN_ID } from '~constants';
+import { useGetMembersForColonyQuery } from '~gql';
+import { useColonyContext } from '~hooks';
+
+export const useOutcomeStep = (motionData: ColonyMotion | null | undefined) => {
```

Comment:
wouldn't that be better?
`motionData: ColonyMotion | null | undefined` -> `motionData?: ColonyMotion | null`

Reply:
motion can be null or undefined as well
<img width="337" alt="Screen Shot 2023-10-25 at 11 54 01" src="https://github.com/JoinColony/colonyCDapp/assets/119587095/f3bf797b-c1db-4cf9-a398-43870a01d844">